source/guide_privatebin.rst

```diff
@@ -0,0 +1,204 @@
+
+.. code-block:: console
+ :emphasize-lines: 2
+
+ [isabell@stardust ~]$ cd /var/www/virtual/$USER/
+ [isabell@stardust isabell]$ wget https://github.com/PrivateBin/PrivateBin/archive/1.2.4.tar.gz
+ […]
+ Saving to: ‘1.2.4.tar.gz’
+
+ 100%[=================================================>] 3,172,029   3.45MB/s   in 0.9s
+
+ 2020-11-22 16:27:44 (8.32 MB/s) - ‘1.2.4.tar.gz’ saved [523648]
+ [isabell@stardust ~]$
+
+Untar the archive to the ``html`` folder and then delete it. Replace the version in the file name with the one you downloaded.
+
+.. code-block:: console
+ :emphasize-lines: 1,2
+
+ [isabell@stardust isabell]$ tar -xzf 1.2.4.tar.gz -C html/ --strip-components=1
+ [isabell@stardust isabell]$ rm 1.2.4.tar.gz
+ [isabell@stardust ~]$
+
+
+Activate the .htaccess file
+-----------------------------
+PrivateBin provides a .htaccess file, which blocks some known robots and link-scanning bots. Activate it by renaming it from ``.htaccess.disabled`` to ``.htaccess``.
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ cd /var/www/virtual/$USER/html
+ [isabell@stardust html]$ mv .htaccess.disabled .htaccess
+ [isabell@stardust ~]$
+
+
+Moving files outside of DocumentRoot
+------------------------------------
+It is recommended_ to move the configuration, data files, templates and PHP libraries outside of your document root.
+To do that, create a Folder  ``privatebin-data`` in ``/var/www/virtual/isabell/`` and move the folders to the new location.
+If not already there, go to the ``html`` directory before running ``mv``.
+
+.. code-block:: console
+
+ [isabell@stardust html]$ mkdir /var/www/virtual/$USER/privatebin-data
+ [isabell@stardust html]$ cd /var/www/virtual/$USER/html
+ [isabell@stardust html]$ mv -t /var/www/virtual/$USER/privatebin-data cfg/ lib/ tpl/ vendor/
+ [isabell@stardust ~]$
+
+Changing index.php
+------------------
+
+Now edit ``/var/www/virtual/$USER/html/index.php``  to inform PrivateBin about to the new location of the folders. 
+
+.. warning:: Replace ``<username>`` with your username!
+
+.. code-block:: php
+ :emphasize-lines: 3
+
+ [...]
+ // change this, if your php files and data is outside of your webservers document root
+ define('PATH', '/var/www/virtual/<username>/privatebin-data/');
+ [...]
+
+Setting permissions
+-------------------
+
+Let's make sure the files and folder have the correct :manual_anchor:`permissions <web-documentroot#permissions>`.
+Use ``find`` in combination with ``chmod`` to change them.
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ cd /var/www/virtual/$USER
+ [isabell@stardust isabell]$ find privatebin-data -type d -exec chmod 755 {} \;
+ [isabell@stardust isabell]$ find privatebin-data -type f -exec chmod 644 {} \;
+ [isabell@stardust ~]$
+
+
+
+Configuration
+=============
+
+Configure your PrivateBin Instance
+----------------------------------
+
+.. note:: You are not forced to change any of the default settings as they are mostly secure.
```

Comment:
"forced" is a bit of a strong word :sweat_smile: 
```suggestion
.. note:: You don't need to change any of the default settings as they are mostly secure.
```

Reply:
I agree :-)